uxarray/grid/geometry.py

```diff
@@ -396,3 +397,152 @@ def _classify_polygon_location(face_edge_cart):
         return "South"
     else:
         return "Equator"
+
+
+def _get_latlonbox_width(latlonbox_rad):
+    """Calculate the width of a latitude-longitude box in degrees. The box
+    should be represented by a 2x2 array in radians and lon0 represent the
+    "left" side of the box. while lon1 represent the "right" side of the box.
```

Comment:
The outputted width appears to be in radians, not degrees.

Reply:
Good find, I have corrected the description